Bottom: c50ad436d83038f2d2b263b932641a68f09e44b2
Top:    6ec6f4295040240d58ed944438d15cbf847a4c65
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-23 17:52:54 +0000

Refresh of remove-draw-call-from-resetfou

---

diff --git a/src/report.c b/src/report.c
index 486ebf4..5dfb3d9 100644
--- a/src/report.c
+++ b/src/report.c
@@ -46,6 +46,7 @@
 #include "macro.h"
 #include "undo.h"
 #include "find.h"
+#include "draw.h"
 
 #ifdef HAVE_LIBDMALLOC
 #include <dmalloc.h>
